tests/examples/examples.cfg

```diff
@@ -1,39 +1,29 @@
-[Example-Stehekin-base-case]
+[Example-Classic-Stehekin-base-case]
+driver = classic
 test_description = Short example simulation of the Stehekin basin.
-global_parameter_file = global_param.STEHE.txt
+global_parameter_file = global_param.classic.STEHE.txt
 check = complete, nonans
 
-[Example-Stehekin-3hewb]
-test_description = Short example simulation of the Stehekin basin using 3hr engergy balance.
-global_parameter_file = global_param.STEHE.3hewb.txt
+[Example-Classic-Stehekin-fewb]
+driver = classic
+test_description = Short example simulation of the Stehekin basin using 1hr full energy balance.
+global_parameter_file = global_param.classic.STEHE.fewb.txt
 check = complete, nonans, energy_balance
-followup = plots, vic2netcdf
-compare = Example-Stehekin-base-case
 
-[Example-Stehekin-wb_output]
+[Example-Classic-Stehekin-wb_output]
+driver = classic
 test_description = Short example simulation of the Stehekin basin checking water balance variables.
-global_parameter_file = global_param.STEHE.wb_output.txt
+global_parameter_file = global_param.classic.STEHE.wb_output.txt
```

Comment:
How is this different from `classic`. The test descriptions should be more precise and there should be a comment that clearly explains the purpose of each test in a one line comment